testing/scenario_app/android/gradle-home/.vpython

```diff
@@ -0,0 +1,11 @@
+#
+# "vpython" specification file for build
+#
+# Pillow is required by com.facebook.testing.screenshot.
+
+python_version: "2.7"
```

Comment:
Any chance we can use 3, or does that need a separate PR?

Reply:
The plugin that compares the screenshots requires `python-2.7`